app/code/core/Mage/Log/Model/Visitor.php

```diff
@@ -158,6 +187,9 @@ public function getUrl()
         return $url;
     }
 
+    /**
+     * @return mixed
```

Comment:
why mixed? I wold expect rather int

Reply:
I've used mixed only in a very few cases, where multiple return types are possble. Mixed/object are on todo list as mentioned in #784.